packages/server/test/index.ts

```diff
@@ -39,6 +37,18 @@ async function createStoreController () {
   })
 }
 
+async function testErrorFormat (t: Test, err: GotError, expectedCode: number) {
+  if (err instanceof HTTPError) {
+    t.equal(err.statusCode, expectedCode, `response code should be a ${expectedCode}`)
```

Comment:
I don't like this check. If err is not HTTPError, the test will pass.

it can be something like 

```ts
t.equal((err as HTTPError).statusCode, expectedCode, `response code should be a ${expectedCode}`)
```

Reply:
I followed your suggestion, but my IDE shows the following error: `TS2304: Cannot find name 'HTTPError'.`

This occurs even with the following import: `import { HTTPError } from 'got'`

Is it because `HTTPError` is not exported from `got`? I was able to suppress this error to get the tests working.